metrics/grafana/tikv_details.json

```diff
@@ -17417,6 +17417,14 @@
               "interval": "",
               "legendFormat": "pending-task",
               "refId": "D"
+            },
+            {
+              "exemplar": true,
+              "expr": "sum(tikv_worker_pending_task_total{k8s_cluster=\"$k8s_cluster\", tidb_cluster=~\"$tidb_cluster.*\", instance=~\"$instance\", name=~\"async-read-worker\"})",
+              "hide": true,
+              "interval": "",
+              "legendFormat": "pending-task-v2",
```

Comment:
What is v2?

Reply:
changed to `async-read-pending-task`

Reply:
legend-format is the name for this metrics, like `{{instance}}-{{type}}`.
![image](https://user-images.githubusercontent.com/6428910/197997644-709531b6-686d-4e92-bb6f-633b1c6439cf.png)